### PR TITLE
RuntimeConfig: Set default variable state to Unspecified

### LINKED ISSUE #3898: Problems accessing Variables properties
using Cloud SDK, I'm able to construct a Configs resource, a Variables resource in it and set a text value but, when I try to access these properties through Cloud Client Libraries for Python, I'm unable to get the values:

```
gcloud beta runtime-config configs variables get-value my-variable \
--config-name=${CONFIG} \
--project=${PROJECT}
```
Returns:
```
my-value
```
```
gcloud beta runtime-config configs variables describe my-variable  \
--config-name=${CONFIG} \
--project=${PROJECT}
```
Returns:
```
atomicName: projects/$PROJECT/configs/$NAME/variables/my-variable
name: my-variable
text: my-value
updateTime: '2017-08-29T15:03:26.359064036Z'
```
And:
```python
from google.cloud import runtimeconfig

client = runtimeconfig.Client()

config = client.config(CONFIG)
print (config.exists())
print(config.get_variable(VARIABLE).exists())
print(config.get_variable(VARIABLE).full_name)
```
Returns:
```
True
True
projects/$PROJECT/configs/$NAME/variables/$VARIABLE
```
But:
```

print(config.get_variable(VARIABLE).state)
print(config.get_variable(VARIABLE).value)
```
Returns:
```
None
None
```

### PR DESCRIPTION
**Pull request by chemelnucfin**
Closes #3898

### CODE REVIEW

File: runtimeconfig/google/cloud/runtimeconfig/variable.py [hunk 1]
```diff
@@ -139,10 +139,13 @@ def state(self):
 
         :rtype: str
         :returns:
-            If set, one of "UPDATED", "DELETED", or
-            "VARIABLE_STATE_UNSPECIFIED", else ``None``.
+            If set, one of "UPDATED", "DELETED", or defaults to
+            "VARIABLE_STATE_UNSPECIFIED".
         """
-        return self._properties.get('state')
```
Comment (tseaver) [inline runtimeconfig/google/cloud/runtimeconfig/variable.py]: You can pass the default value as a second argument to `dict.get`, e.g.:

```python
        return self._properties.get('state', STATE_UNSPECIFIED)
```

Comment (chemelnucfin) [reply runtimeconfig/google/cloud/runtimeconfig/variable.py]: fixed, PTAL

File: runtimeconfig/google/cloud/runtimeconfig/variable.py [hunk 2]
```diff
@@ -142,9 +142,7 @@ def state(self):
             If set, one of "UPDATED", "DELETED", or defaults to
             "VARIABLE_STATE_UNSPECIFIED".
         """
-        value = self._properties.get('state')
-        if value is None:
-            value = STATE_UNSPECIFIED
+        value = self._properties.get('state', STATE_UNSPECIFIED)
```
Comment (tseaver) [inline runtimeconfig/google/cloud/runtimeconfig/variable.py]: No need to assign a local variable, just return it.

**Comment by chemelnucfin** [comment]
fixed.  should be good to go on green.

**Pull request merged**